src/tracker/index.js

```diff
@@ -233,21 +233,34 @@
   };
 
   const track = (obj, data) => {
+    let identity;
+    try {
+      const parsedIdentity = JSON.parse(localStorage.getItem('umami.identity'));
```

Comment:
We can't use localStorage for privacy reasons. We might have to rely on umami.identify and change the signature to `umami.identify(id, data)`.